src/js/relayclient/Environments.js

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
its a good idea to define "Environments" outside of actual code and tests.
its bad idea to scatter `istanbul` everywhere - it should be a single value here, at the environment
(e.g. all should access `Environments.default`